ci/test.sh

```diff
@@ -182,6 +182,21 @@ main() {
 
         rm -rf "${td}"
     fi
+
+    # special tests for a shared C runtime, since we disable the shared c++ runtime
```

Comment:
This doesn't need a check for C++ support since it's for a shared C runtime. The C++ runtime will always be static, and we've tested that above.